go/logger/standard.go

```diff
@@ -193,8 +193,13 @@ func (log *Standard) Error(fmt string, arg ...interface{}) {
 	}
 }
 
+// Errorf is a deprecated method until all the formatting methods are fixed (use Error instead)
```

Comment:
To clarify the logging api in client/go/logger is:

```
Debug 
Info
Notice
Warning
Error
Errorf
```

All take formatting string as first param despite not having `f` at the end, which kind of breaks the convention.

We could do a mass search replace to change Debug to Debugf, Info to Infof etc which would fix everything but...